src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
We explored several questions here:
* What's my idea regarding next steps (on anti_lmr7 pass/fail)?  (I don't know what would be uncovered during cause analysis.)
* What's my motivation (why is it important) for searching for a similar patch?

Anyway...
Sorry, I wasn't quoting, I just didn't have a term for our earlier difference in perspective & while I see positive results (both Elo and scaling) at present I don't know & am trying to understand it before making a value judgment.  Even earlier (my plan based on `anti_lmr7` results) didn't state anything about rejecting a patch (but that I intended to do further analysis to try to better understand this patch) & I had also stated I'd likely merge this PR.  Anyway, I guess we'll see how next steps go...